LICENSE

```diff
@@ -49,6 +51,26 @@ For these three licenses, full length texts are available hereafter:
    POSSIBILITY OF SUCH DAMAGE.
 
 
+                            Synopsys MIT License
```

Comment:
Move below Apache License.
(Since we don't have rule for this, I'd propose to list them in order of frequency of use)